linter_exclusions.yml

```diff
@@ -1627,6 +1627,38 @@ network vhub connection create:
     allow_remote_vnet_to_use_hub_vnet_gateways:
       rule_exclusions:
       - option_length_too_long
+network vpn-gateway connection vpn-site-link-conn add:
+  parameters:
+    use_local_azure_ip_address:
+      rule_exclusions:
+      - option_length_too_long
+    use_policy_based_traffic_selectors:
+      rule_exclusions:
+      - option_length_too_long
+    vpn_connection_protocol_type:
+      rule_exclusions:
+      - option_length_too_long
+    vpn_link_connection_mode:
+      rule_exclusions:
+      - option_length_too_long
+    vpn_site_link_conn_name:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
please use '--name' and '-n' for this argument under `network vpn-gateway connection vpn-site-link-conn` command group